src/Elmish.WPF/Binding.fs

```diff
@@ -209,24 +210,40 @@ module Binding =
       >> mapModel ValueOption.ofOption
       >> mapMsg ValueOption.toOption
 
+      
+  module Cmd =
 
-  module SubModel =
```

Comment:
Why is the diff complicated starting here?  Did you change the order of the modules?

Reply:
No, I added a `Cmd` module and the diff is messed up.

Reply:
The `Cmd` module used to be in `BindingInternal.fs` but now is here because it is dealing with `Binding` types.

Reply:
Can change the order of the modules or split the commit so the diff isn't messed up?

Reply:
Resolved by messing with whitespace